joomla/joomla-cms
# Correct archived article links in search results

Pull Request for Issue #22395.

### Summary of Changes

Corrects links to archived articles in search results.

### Testing Instructions

Make sure `Archived Articles` option is enabled in `Search - Content` plugin.

Create an article. Set status to archived.
Search for the article using standard search (`com_search`).
Inspect link to article in search results.

### Expected result

Link opens article, e.g. `http://localhost/index.php/site-map/articles/9-uncategorised/67-whats-new-in-15`

### Actual result

Link opens archived view, e.g. `http://localhost/index.php/using-joomla/extensions/components/content-component/archived-articles/2019?view=archive&month=5`

### Documentation Changes Required

No.
